conans/client/cmd/download.py

```diff
@@ -14,9 +14,12 @@ def download(app, ref, package_ids, remote, recipe, recorder, remotes):
     hook_manager.execute("pre_download", reference=ref, remote=remote)
 
     try:
-        ref = remote_manager.get_recipe(ref, remote)
+        _, ref = remote_manager.get_recipe_manifest(ref, remote)
```

Comment:
We should probably avoid file transfers just to get the revision. Isn't there any other endpoint better than the manifest. I think in Conan 2.0 the manifest retrieval can actually be dropped in whole codebase.